CHANGELOG.md

```diff
@@ -1,36 +1,52 @@
 ## 0.88.0
-* BREAKING CHANGE: Realm has now to be installed as a Gradle plugin.
-* BREAKING CHANGE: DynamicRealm.executeTransaction() now directly throws any RuntimeException instead of wrapping it in a RealmException (#1682).
-* BREAKING CHANGE: DynamicRealm.executeTransaction() now throws IllegalArgumentException instead of silently accepting a null Transaction object.
-* BREAKING CHANGE: String setters now throw IllegalArgumentException instead of RealmError for invalid surrogates.
-* BREAKING CHANGE: DynamicRealm.distinct()/distinctAsync() and Realm.distinct()/distinctAsync() now throw IllegalArgumentException instead of UnsupportedOperationException for invalid type or unindexed field.
-* BREAKING CHANGE: All thread local change listeners are now delayed until the next Looper event instead of being triggered when committing.
-* BREAKING CHANGE: Removed RealmConfiguration.getSchemaMediator() from public API which was deprecated in 0.86.0. Please use RealmConfiguration.getRealmObjectClasses() to obtain the set of model classes (#1797).
-* BREAKING CHANGE: Realm.migrateRealm() throws a FileNotFoundException if the Realm file doesn't exist.
-* BREAKING CHANGE: It is now required to unsubscribe from all Realm RxJava observables in order to fully close the Realm (#2357).
+
+### Breaking changes
```

Comment:
Why not a `Breaking Changes` and a `Deprecations` sections to prevent repeating the first word?